From dbb9399493704d8d9428828161a0548e1aa379c4 Mon Sep 17 00:00:00 2001
From: Kei Okada <kei.okada@gmail.com>
Date: Sat, 6 Feb 2016 12:33:49 +0000
Subject: [PATCH] fix wrong robot_self_filter_LIBRAIRES, this only happens on
 utopic and vivid

---
 CMakeLists.txt |    7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index fdd356d..00f317e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -203,6 +203,13 @@ add_library(${PROJECT_NAME} SHARED ${jsk_perception_nodelet_sources}
   src/image_utils.cpp
   src/histogram_of_oriented_gradients.cpp
   )
+target_link_libraries(${PROJECT_NAME} ${catkin_LIBRARIES} yaml-cpp)
+set(robot_self_filter_LIBRARIES2)
+foreach(lib ${robot_self_filter_LIBRARIES})
+  string(REPLACE ":/" "/" lib ${lib})
+  list(APPEND robot_self_filter_LIBRARIES2 ${lib})
+endforeach()
+set(robot_self_filter_LIBRARIES ${robot_self_filter_LIBRARIES2})
 target_link_libraries(${PROJECT_NAME} ${catkin_LIBRARIES} ${robot_self_filter_LIBRARIES} yaml-cpp)
 if(robot_self_filter_FOUND)
   include_directories(${robot_self_filter_INCLUDE_DIRS})
-- 
1.7.9.5

